app/views/donors/_edit.html.erb

```diff
@@ -0,0 +1,46 @@
+<%= form_for(@donor) do |f| %>
+
+<div>
+  <%= link_to @donor do %>
+  <i class="fa fa-close fa-lg"></i>
+  <% end %>
+</div>
+
+<div class="row">
+  <div class="col-xs-6 col-sm-6 col-md-6 col-lg-6">
+    <div class="form-group">
+      <label for="donorDocs" class="control-label">Name</label>
+      <%= f.text_field :name, class: "form-control" %>
+    </div>
+
+    <div class="form-group">
+      <label for="donorDocs" class="control-label">NRIC/UEM Number</label>
```

Comment:
We changed the label to be just "NRIC/UEM" in Maria's code, so let's use that consistently throughout the application. 😀